src/EventDataStore/DbalEventDataStore.php

```diff
@@ -0,0 +1,90 @@
+<?php
+declare(strict_types=1);
+
+namespace Landingi\EventStoreBundle\EventDataStore;
+
+use Doctrine\DBAL\Connection;
+use Landingi\EventStoreBundle\Event\AccountName;
+use Landingi\EventStoreBundle\Event\AccountUuid;
+use Landingi\EventStoreBundle\Event\UserEmail;
+use Landingi\EventStoreBundle\Event\UserRole;
+use Landingi\EventStoreBundle\Event\UserUuid;
+use Landingi\EventStoreBundle\EventDataStore;
+use RuntimeException;
+
+final class DbalEventDataStore implements EventDataStore
```

Comment:
Ja bym się zastanowił, czy Ciebie tutaj bardziej nie interesuje zrobienie cache'a na podstawie dekoratora.

Wtedy miałbyś:
* EventDataStore interfejs
* DbalEventDataStore implementacje
* MemoryCacheEventDataStore albo InMemoryCachedEventDataStore dekorator

Reply:
W zasadzi chodzi mi o to, że można to dopracować.. i przetestować XD

Reply:
no a to spoko pomysł, też myśałem o tym ale uznałem za overkill i wolałem zaczekać XD

Reply:
@Ferror Btw, cache? Tobie na pewno chodzilo o cache, czy o wspolny interface dla wielu zestawow zrodel danych?

Reply:
o Cache, ale dekoratorem można by to było ograć.. no tutaj jest wiele możliwości XD